chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/DeletedRoom.kt

```diff
@@ -0,0 +1,17 @@
+package com.pusher.chatkit.rooms.state
+
+import com.pusher.chatkit.state.ChatkitState
+import org.reduxkotlin.reducerForActionType
+
+internal data class DeletedRoom(
```

Comment:
Room here is the subject and it sounds better when it's put first. That naming has been suggested for those events by @mdpye and was put in the [API sketch](https://github.com/pusher/chatkit-android/pull/174/files#diff-7b362c0cc548b3a5a696fdd71ca57fbbR28) we've paired on. Joined/Left Room are different as for them the current user is the (implicit) subject.

Reply:
👍 have refactored to ensure this PR matches the v2 sketch.